### PR TITLE
Bug 1125801 - [Costcontrol] Javascript error on startup

### PR DESCRIPTION
**Pull request by gitmai**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gitmai Marina (gitmai)](https://github.com/gitmai) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d28ae1496998)

File: apps/costcontrol/js/app.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ var CostControlApp = (function() {
 
   var costcontrol,
       initialized = false,
+      airplanemodeActive = false,
```
Comment (lodr) [inline apps/costcontrol/js/app.js]: nit: rename to `isAirplaneModeActive` please.

File: apps/costcontrol/js/app.js [hunk 2]
```diff
@@ -242,7 +244,7 @@ var CostControlApp = (function() {
 
   window.addEventListener('airplaneModeDisabled',
     function _onAirplanemodeDisabled(evt) {
-      if (evt.detail && evt.detail.serviceId === 'data') {
+      if (airplanemodeActive && evt.detail && evt.detail.serviceId === 'data') {
         waitForSIMReady(startApp);
       }
     });
```
Comment (lodr) [inline apps/costcontrol/js/app.js]: nit: please, add a breakline before the close bracket and align it below the `w` of `window.addEventListener()`

**Comment by try-server-hook** [comment]
[gitmai Marina (gitmai)](https://github.com/gitmai) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a22c33f7a815)

**Comment by try-server-hook** [comment]
[gitmai Marina (gitmai)](https://github.com/gitmai) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=40503617c391)

**Pull request merged**